### PR TITLE
pin docker-compose <1.29 as it requires docker v5

### PR DESCRIPTION
**Pull request by ryan109**
## Purpose of PR
At the moment if you install kubetools it will install `1.29.*`, which breaks due to:
```
pkg_resources.ContextualVersionConflict: (docker 4.4.4 (/Users/ryanmccorry/.pyenv/versions/3.7.6/lib/python3.7/site-packages), Requirement.parse('docker[ssh]>=5'), {'docker-compose'})
```
## Related links

### CODE REVIEW

**Review by edmond-edited — COMMENTED** [review]
Ignore, misread

**Review by edmond-edited — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -59,7 +59,8 @@ def get_readme_content():
             'kubernetes',
             'tabulate<1',
             # compose v2 has broken container naming
-            'docker-compose<2',
+            # and compose 1.29 requires docker >5
```
Comment (mattalexanderhill) [inline setup.py:62]: Is there any reason to specify `docker<5`? 

The only breaking changes listed are dropping python 2.7 and 3.5 support, are they deal breakers?

https://docker-py.readthedocs.io/en/stable/change-log.html

Comment (mattalexanderhill) [reply setup.py:62]: This is where it was pinned last time: https://github.com/EDITD/kubetools/commit/3a668b40cd68c0b60b5f5d1922bd290c3cc14618

I think before `docker==5.0` was released https://pypi.org/project/docker/#history

**Review by gchazot — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (ttecles) [reply setup.py:62]: I guess this is a safety guard (pinning to a lower mayor version) but given that this is a package I would also try to loosen the dependencies as much as possible. And seems @mattalexanderhill is right.

@ryan109 @gchazot @edmond-edited what do you think about this?

Comment (gchazot) [reply setup.py:62]: needs testing...
but otherwise no reason ~to pin docker~ **NOT** to pin docker

**Comment by gchazot** [comment]
Is this still necessary? AFAIK kubetools is not causing any issues at the moment without it...

**Comment by ryan109** [comment]
Honestly can't remember, I can close it for now and reopen if we see issues.

**Pull request closed without merging**